avaframe/ana5Utils/distanceTimeAnalysisCfg.ini

```diff
@@ -0,0 +1,50 @@
+### Config File - This file contains the main settings for the range time diagram in distanceTimeAnalysis
+## Copy to  local_distanceTimeAnalysisCfg.ini and set you parameters
+# This file is part of Avaframe.
+
+
+[GENERAL]
+# result type used for creating range time diagram e.g. FD, FV, ....
+rangeTimeResType = FD
+# time steps used for creating diagram
+tSteps = 0:1
+# threshold - below this result data is ignored - used to create mask
+# average value created of where there is avalanche flow (threshold > 0.0)
+# this is default set to 1.e-5 to account for any spurious values
+# also a different threshold can be set e.g. if only regions with > 0.5m FD are of interest
+thresholdResult = 1.e-5
+
+#+++++range time diagram with radar field of view +++++
+# width of range gates in meters
+rgWidth = 5.
+# aperture angle in degrees
+aperture = 15
+
+# radar location field of view points min (x0, y0) and point giving direction (x1, y1)
+# format: x0|x1|y0|y1 needs to be in same projection as DEM
+radarLocation =
```

Comment:
Add a check, if the radar location is within the valid area of the DEM. And tell the user, that the DEM is not suited for this radar location. Perhaps help him with generating the radarFieldOfViewPlotDEM (which does not need a z-value and any x,y coordinate value outside the DEM should work) already.

Reply:
an error occurs with message: 'Radar location is invalid, required format x0|x1|y0|y1 not: %s' %
            cfg['GENERAL']['radarLocation'])

Reply:
if (np.any(radarFov[0] < np.amin(X)) or np.any(radarFov[0] > np.amax(X))  or
        np.any(radarFov[1] < np.amin(Y))  or np.any(radarFov[1] > np.amax(Y))):
        message = 'Radar location outside of DEM, x= %.2f, %.2f, y= %.2f, %.2f' % (radarFov[0][0],
            radarFov[0][1], radarFov[1][0], radarFov[1][1])
        log.error(message)
        raise AssertionError(message)